.claude/skills/extract-common-component/SKILL.ko.md

```
@@ -0,0 +1,344 @@
+grep -rh 'className="[^"]\{30,\}"' src/app/ src/components/pages/ --include="*.tsx" \
+  | sed 's/.*className="\([^"]*\)".*/\1/' \
+  | sort | uniq -d -c | sort -rn | head -20
+```
+
+교차 검증: 2개 이상의 파일에서 나타나는 `className` 문자열이 후보 지문.
+
+#### 0c. 후보 랭킹
+
+각 후보에 점수 부여:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)